src/workflow/restore.c

```diff
@@ -47,6 +49,10 @@ bool workflow_restore_backup(const RestoreBackupRequest* restore_request)
     if (!memory_set_initialized()) {
         return false;
     }
+    if (!workflow_confirm_time(restore_request->timestamp, restore_request->timezone_offset)) {
+        return false;
+    }
+    securechip_u2f_counter_set(restore_request->timestamp);
```

Comment:
error not checked (can also be explicit ignore).

maybe add USE_RESULT to securechip.h

Reply:
ack